src/apps/vpn/ui/screens/getHelp/developerMenu/ViewUiTesting.qml

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.5
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import Mozilla.Shared 1.0
+import components 0.1
+
+MZViewBase {
+    _menuTitle: "UI Testing"
+    _viewContentData: ColumnLayout {
+        id: featureListHolder
+        spacing: MZTheme.theme.windowMargin
+        Layout.fillWidth: true
+
+        MZSettingsItem {
+            settingTitle: "MZStepProgressBar"
+            imageRightSrc: "qrc:/nebula/resources/chevron.svg"
+            imageRightMirror: MZLocalizer.isRightToLeft
+            onClicked: getHelpStackView.push(progressBar)
+            Layout.leftMargin: MZTheme.theme.windowMargin / 2
+            Layout.rightMargin: MZTheme.theme.windowMargin / 2
+        }
+    }
+
+    Component {
+        id: progressBar
+        MZViewBase {
+            _menuTitle: "MZStepProgressBar"
+            _viewContentData: ColumnLayout {
+                Layout.leftMargin: 36
+                Layout.rightMargin: 36
+
+                spacing: 32
+
+                MZStepProgressBar {
+                    id: stepProgress
+                    Layout.alignment: Qt.AlignHCenter
+                    Layout.fillWidth: true
+
+
+                    model: ListModel {
+                        id: stepProgressModel
+
+                        //Hacky workaround since QML list models can't take in scripts (the MZI18n localization script) as property values
+                        //This circumvents us having to make a C++ class for the progress bar delegates and a model for the progress bar as a whole
+                        //(i.e. navigationbarmodel.h/cpp and navigationbarbutton.h/cpp)
+                        //reference: https://stackoverflow.com/a/33161093
+                        property bool completed: false
+                        property string firstString: MZI18n.OnboardingProgressBarDataUse
```

Comment:
What about handling this the same way as we do for `VPNSegmentedNavigation` ? 
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/308692c7ee9d3bda41d8d6e4cd9af9dadc83c659/nebula/ui/components/MZSegmentedToggle.qml#L146
And then...
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/308692c7ee9d3bda41d8d6e4cd9af9dadc83c659/src/apps/vpn/ui/screens/home/ViewServers.qml#L75-L84

Reply:
That...is a good idea. Thanks